samples/test_azure_event_hubs_receive.py

```diff
@@ -128,6 +128,40 @@ def test_event_hubs_client_receive_sync(live_eventhub_config):
     log.info("Finished receiving")
 
 
+def test_event_hubs_client_receive_with_runtime_metric_sync(live_eventhub_config):
+    uri = "sb://{}/{}".format(live_eventhub_config['hostname'], live_eventhub_config['event_hub'])
+    sas_auth = authentication.SASTokenAuth.from_shared_access_key(
+        uri, live_eventhub_config['key_name'], live_eventhub_config['access_key'])
+
+    source = "amqps://{}/{}/ConsumerGroups/{}/Partitions/{}".format(
+        live_eventhub_config['hostname'],
+        live_eventhub_config['event_hub'],
+        live_eventhub_config['consumer_group'],
+        live_eventhub_config['partition'])
+
+    receiver_runtime_metric_symbol = b'com.microsoft:enable-receiver-runtime-metric'
+    symbol_array = [types.AMQPSymbol(receiver_runtime_metric_symbol)]
+    desired_capabilities = utils.data_factory(types.AMQPArray(symbol_array))
+
+    with uamqp.ReceiveClient(source, auth=sas_auth, debug=False, timeout=50, prefetch=50,
+                             desired_capabilities=desired_capabilities) as receive_client:
+        log.info("Created client, receiving...")
+        with pytest.raises(ValueError):
+            batch = receive_client.receive_message_batch(max_batch_size=100)
+        batch = receive_client.receive_message_batch(max_batch_size=10)
+        log.info("Got batch: {}".format(len(batch)))
+        assert len(batch) <= 10
+        for message in batch:
+            annotations = message.annotations
+            delivery_annotations = message.delivery_annotations
+            log.info("Sequence Number: {}".format(annotations.get(b'x-opt-sequence-number')))
+            log.info("Last enqueued sequence number: {}".format(delivery_annotations.get(b'last_enqueued_sequence_number')))
+            log.info("Last enqueued offset: {}".format(delivery_annotations.get(b'last_enqueued_offset')))
+            log.info("Last enqueued time utc: {}".format(delivery_annotations.get(b'last_enqueued_time_utc')))
+            log.info("Runtime info retrieval time utc: {}".format(delivery_annotations.get(b'runtime_info_retrieval_time_utc')))
```

Comment:
These log statements should be assert statements:
`assert b'last_enqueued_sequence_number' in annotations`

Reply:
test updated